integration/tests/conftest.py

```diff
@@ -205,13 +205,28 @@ def erc20_spl_mintable(web3_client: NeonChainWeb3Client, faucet, sol_client, sol
 
 
 @pytest.fixture(scope="function")
-def new_account(web3_client, faucet, eth_bank_account):
-    yield web3_client.create_account_with_balance(faucet, bank_account=eth_bank_account)
+def new_account(web3_client, sol_client, faucet, eth_bank_account,
+                web3_client_sol, pytestconfig, solana_account):
+    account = web3_client.create_account_with_balance(faucet, bank_account=eth_bank_account)
+    sol_client.deposit_wrapped_sol_from_solana_to_neon(solana_account,
+                                                       account,
+                                                       web3_client_sol.eth.chain_id,
+                                                       pytestconfig.environment.evm_loader)
+    yield account
 
 
 @pytest.fixture(scope="class")
-def class_account(web3_client, faucet, eth_bank_account):
-    yield web3_client.create_account_with_balance(faucet, bank_account=eth_bank_account)
+def class_account(web3_client, faucet, eth_bank_account, solana_account, sol_client, web3_client_sol, pytestconfig):
+    account = web3_client.create_account_with_balance(faucet, bank_account=eth_bank_account)
+    sol_client.request_airdrop(solana_account.public_key, 5 * LAMPORT_PER_SOL)
```

Comment:
this airdrop can't work in devnet with bank account. For 4 threads we don't have 20 sols in devnet